core/src/main/java/org/apache/calcite/sql/SqlCreate.java

```diff
@@ -26,30 +26,49 @@
 public abstract class SqlCreate extends SqlDdl {
 
   /** Whether "OR REPLACE" was specified. */
```

Comment:
nit: The comment block need to be change to reflect the change from boolean to enum.